transaction.go

```diff
@@ -130,6 +130,12 @@ type TransactionRequest struct {
 	LineItems           TransactionLineItemRequests `xml:"line-items,omitempty"`
 }
 
+type TransactionRefundRequest struct {
+	XMLName string   `xml:"transaction"`
+	Amount  *Decimal `xml:"amount"`
+	OrderId string   `xml:"order-id,omitempty"`
```

Comment:
Can we have this parameter `OrderID`? There's a mix of casings in this package but trying to make new ones consistent with Go standards.